app/tests/Creating Tests.md

```diff
@@ -0,0 +1,23 @@
+# Basics
```

Comment:
Any reason this shouldn't be in the "Development" section of the website docs?

Reply:
Honest answer? Because I'm not sure if people are reading the docs. I figured it would be right in their face here

Reply:
Moved and updated to link to other docs